api/pom.xml

```diff
@@ -50,7 +48,8 @@
         <maven-enforcer-plugin.version>3.0.0-M3</maven-enforcer-plugin.version>
         <build-helper-maven-plugin.version>3.1.0</build-helper-maven-plugin.version>
         <maven-project-info-reports-plugin.version>3.0.0</maven-project-info-reports-plugin.version>
-        <jakarta.transaction-api.version>2.0.0</jakarta.transaction-api.version>
+        <jakarta.transaction-api.version>2.0.1-RC1</jakarta.transaction-api.version>
+        <jakarta.cdi-api.version>3.0.1</jakarta.cdi-api.version>
```

Comment:
There are no changes to the EJB API for Jakarta EE 10, so why is a dependency on CDI being added?

Reply:
Also, I would assume this would need to be for EJB 4.0.1-RC1 until transactions releases 2.0.1.

Reply:
JTA has a transitive dependency on CDI. Building without the dependency produces:
```
[INFO] Compiling 82 source files to /Users/starksm/Dev/JBoss/Jakarta/rh-ejb/api/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] module not found: jakarta.cdi
```

Reply:
> JTA has a transitive dependency on CDI. Building without the dependency produces:
> 
> ```
> [INFO] Compiling 82 source files to /Users/starksm/Dev/JBoss/Jakarta/rh-ejb/api/target/classes
> [INFO] -------------------------------------------------------------
> [ERROR] COMPILATION ERROR : 
> [INFO] -------------------------------------------------------------
> [ERROR] module not found: jakarta.cdi
> ```

Already approved
- #141 
compiles fine, and does not add CDI dependency.

Reply:
I was going to comment on the dependency on cdi as well. I think it will automatically pull in the transitive dependencies.

Reply:
It is not provided to the compiler because the pom level dependency of transactions on CDI is provided scope, and maven does not include this in the dependency graph:
```
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ jakarta.ejb-api ---
[INFO] jakarta.ejb:jakarta.ejb-api:jar:4.0.1-SNAPSHOT
[INFO] \- jakarta.transaction:jakarta.transaction-api:jar:2.0.1-RC1:compile
[INFO] ------------------------------------------------------------------------
```